src/index.html

```diff
@@ -2,15 +2,56 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta  name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+    <form 
+      class="search" 
+      data-qa="big" 
+      action="http://localhost:8080" 
+      method="post"
+    >
+      <label>
+        <img 
+          alt="icon" 
+          class="icon" 
```

Comment:
Don't you need to add modifiers for the big input as well? 
```suggestion
          class="icon icon--big" 
```